<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoSocket</class>
 <widget class="QMainWindow" name="LoSocket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoSocket</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox { 
	border: 1px solid gray; 
	border-radius: 4px;
	margin-top: 8px;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
}
QTabWidget::pane { /* The tab widget frame */
    border-top: 2px solid #C2C7CB;
	border-bottom: 2px solid #C2C7CB;
}
QTabWidget::tab-bar {
    left: 5px; /* move to the right by 5px */
}
QTabBar::tab {
	border-top: 2px solid #C2C7CB;
	border-left: 2px solid #C2C7CB;
	border-right: 2px solid #C2C7CB;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 26ex;
    padding: 2px;
}
QTabBar::tab:selected, QTabBar::tab:hover {    
	background-color: rgb(255, 255, 255);
}
QTabBar::tab:selected {
    border-color: #9B9B9B;
    border-bottom-color: #C2C7CB; /* same as pane color */
}
QTabBar::tab:!selected {
    margin-top: 2px; /* make non-selected tabs look smaller */
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,0,0">
    <item>
     <widget class="QTabWidget" name="SocketPages">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,1,0">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Attach</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="cbAttachCR">
           <property name="text">
            <string>CR</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbAttachLF">
           <property name="text">
            <string>LF</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbAttachZR">
           <property name="text">
            <string>ZERO</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grpType">
        <property name="title">
         <string>Type</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="rbnText">
           <property name="text">
            <string>Text</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">rbngType</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbnHex">
           <property name="text">
            <string>Hex</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">rbngType</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbnFile">
           <property name="text">
            <string>File</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">rbngType</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Send</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QCheckBox" name="cbSend2All">
             <property name="text">
              <string>Send to All Page</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QCheckBox" name="cbEnter">
             <property name="text">
              <string>Enter to Send &amp;&amp;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="cbAutClear">
             <property name="text">
              <string>Aut Clear</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QCheckBox" name="cbAutSend">
             <property name="text">
              <string>Aut Send /</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sbAutSend">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbnMS">
             <property name="text">
              <string>ms</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">rbngTime</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbnS">
             <property name="text">
              <string>s</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">rbngTime</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbnMin">
             <property name="text">
              <string>min</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">rbngTime</string>
             </attribute>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="grpBns">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="bnSend">
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bnClear">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bnFile">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>File</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="TxData"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionChinese"/>
    <addaction name="actionEnglish"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuLanguage"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionChinese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Chinese</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>rbnFile</sender>
   <signal>toggled(bool)</signal>
   <receiver>bnFile</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>553</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>556</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbAutSend</sender>
   <signal>toggled(bool)</signal>
   <receiver>rbnS</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbAutSend</sender>
   <signal>toggled(bool)</signal>
   <receiver>rbnMS</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbAutSend</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbAutSend</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbAutSend</sender>
   <signal>toggled(bool)</signal>
   <receiver>rbnMin</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbAutSend</sender>
   <signal>toggled(bool)</signal>
   <receiver>grpType</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbAutSend</sender>
   <signal>toggled(bool)</signal>
   <receiver>grpBns</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChinese</sender>
   <signal>triggered()</signal>
   <receiver>LoSocket</receiver>
   <slot>onLanguageChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEnglish</sender>
   <signal>triggered()</signal>
   <receiver>LoSocket</receiver>
   <slot>onLanguageChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>LoSocket</receiver>
   <slot>onAboutShown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bnSend</sender>
   <signal>clicked()</signal>
   <receiver>LoSocket</receiver>
   <slot>onDataSent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbAutSend</sender>
   <signal>toggled(bool)</signal>
   <receiver>LoSocket</receiver>
   <slot>onAutSendChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>545</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbnHex</sender>
   <signal>toggled(bool)</signal>
   <receiver>LoSocket</receiver>
   <slot>onTxtSendConverted(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbEnter</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbAutClear</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>517</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>517</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bnClear</sender>
   <signal>clicked()</signal>
   <receiver>TxData</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bnFile</sender>
   <signal>clicked()</signal>
   <receiver>LoSocket</receiver>
   <slot>onFileOpened()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>751</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onLanguageChanged()</slot>
  <slot>onAboutShown()</slot>
  <slot>onDataSent()</slot>
  <slot>onAutSendChanged(bool)</slot>
  <slot>onTxtSendConverted(bool)</slot>
  <slot>onFileOpened()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="rbngType"/>
  <buttongroup name="rbngTime"/>
 </buttongroups>
</ui>
